### PR TITLE
update

### PR DESCRIPTION
**Pull request by pheobeayo**
update

### CODE REVIEW

File: tsconfig.json
Comment (CVamsi27) [inline tsconfig.json:1]: convert all .jsx files to .tsx
Remove change in this file

File: src/components/NavBar.tsx
Comment (CVamsi27) [inline src/components/NavBar.tsx:1]: Pull changes from main branch
```
git pull https://github.com/NovaNex-Hub/website.git main
```
Format using Prettier, use
```
pnpm exec prettier . --write
```

**Comment by CVamsi27** [comment]
@pheobeayo Added Comments

File: tsconfig.json
Comment (CVamsi27) [reply tsconfig.json:1]: Use Next.js 14 app router instead of the old pages router

**Comment by CVamsi27** [comment]
Update title and description

**Comment by pheobeayo** [comment]
approve the next PR

**Pull request closed without merging**